### DISCUSSION TITLE
Notification messages go out of the editor window.

### DISCUSSION THREAD

**Discussion by elgar328** [LazyVim/LazyVim#1404] [Q&A] [answered]
I am new to neovim and lazyvim.
I installed lazyvim and installed all plugins related to rust according to the contents of [this page](https://www.lazyvim.org/extras/lang/rust).
I put the full-spec of all plugins in the nvim/lua/plugins/for_rust.lua file as shown below.

<img width="1144" alt="스크린샷 2023-09-02 오전 12 58 55" src="https://github.com/LazyVim/LazyVim/assets/93251045/b766e968-0e66-4f78-aa60-a2eaa58e1327">

When I write Rust code, a notification message appears as shown below. However, it is impossible to read the content behind the message beyond the width of the editor window.

<img width="667" alt="스크린샷 2023-09-02 오전 12 55 20" src="https://github.com/LazyVim/LazyVim/assets/93251045/43ce0bb0-a7f1-4b94-87c7-435b8d4fd662">

I wonder if the problem is caused by something I configured incorrectly.
And I would like to know which plugin this issue is related to.
And I'm curious about the name of that type of message. I couldn't search for it accurately because I didn't know its name.

**Comment by dpetka2001** [comment, marked as answer]
You can use `<leader>cd` to open the diagnostic in a float window to see the whole message. The way the diagnostic is shown in your screenshot, is because of `virtual_text`. However, I'm not aware if you can use wrapping for the diagnostic message, so that it spans multiple lines if it's longer than the width of the window. You can also use `<leader>xx` for `Trouble Document Diagnostics`. `Telescope diagnostics` have an issue right now that is being worked on [here](https://github.com/nvim-telescope/telescope.nvim/pull/2662) and as a workaround you would have to manually call `Telescope diagnostics severity_bound=ERROR` to see the diagnostics in the Telescope picker. Pick what fits best your workflow for the way to see the diagnostics (I personally prefer `Trouble Document Diagnostics` with the keybindings that it provides to go to next diagnostic from the buffer that I'm in).

**Reply by elgar328** [reply]
Thank you very much! It was a great help.
